test/Microsoft.AspNet.Authentication.Test/OAuthBearer/OAuthBearerMiddlewareTests.cs

```diff
@@ -27,6 +26,8 @@ public class OAuthBearerMiddlewareTests
         {
             var server = CreateServer(options =>
             {
+                options.AutomaticAuthentication = true;
```

Comment:
In this rather uncommon scenario (at least, I hope), I don't see why cookie authentication should be automatic by default as it is far more prone to XSRF attacks than bearer authentication.